### PR TITLE
FIO | push performance results into the dashboard

### PR DESCRIPTION
**Pull request by Avilir**
Signed-off-by: Avi Layani <alayani@redhat.com>

### CODE REVIEW

**Comment by Avilir** [comment]
until #5277 will fix, i am running the validation locally.

**Review by ypersky1980 — CHANGES_REQUESTED** [review]
@Avilir  , Can you use class ResultsAnalyse from ocs_ci.ocs.perfresult  in this test as well, instead of FIOResultsAnalyse ( not needed after you use ResultsAnalyse from ocs_ci.ocs.perfresult) ?

**Comment by Avilir** [comment]
> @Avilir , Can you use class ResultsAnalyse from ocs_ci.ocs.perfresult in this test as well, instead of FIOResultsAnalyse ( not needed after you use ResultsAnalyse from ocs_ci.ocs.perfresult) ?

since this object have more methods, which is not part of `ResultsAnalyse` and not needed in other objects, i created a new object.

**Review by ypersky1980 — APPROVED** [review]

File: tests/e2e/performance/io_workload/test_fio_benchmark.py [hunk 1]
```diff
@@ -391,16 +398,22 @@ def test_fio_workload_simple(self, interface, io_pattern):
             io_pattern (str): the I/O pattern to do - random / sequential
 
         """
+        # Getting the full path for the test logs
+        self.full_log_path = get_full_test_logs_path(cname=self)
+        self.results_path = get_full_test_logs_path(cname=self)
+        self.full_log_path += f"-{interface}-{io_pattern}"
+        log.info(f"Logs file path name is : {self.full_log_path}")
+        log.info(f"reslut path is : {self.results_path}")
```
Comment (keemano) [inline tests/e2e/performance/io_workload/test_fio_benchmark.py:406]: spelling fix:- "result"

**Comment by Avilir** [comment]
from the PR validation job : 
```
2021-12-29 10:57:33  08:57:33 - MainThread - ocs_ci.utility.perf_dash.dashboard_api - INFO - Test results pushed to the DB!
2021-12-29 10:57:33  08:57:33 - MainThread - ocs_ci.ocs.perftests - INFO - Full results report can be found at : http://10.0.78.167:8080/index.php?version1=14&build1=24&platform1=8&az_topology1=3&test_name%5B%5D=1&submit=Choose+options
2021-12-29 10:57:33  08:57:33 - MainThread - ocs_ci.utility.perf_dash.dashboard_api - INFO - Closing the DB connection
2021-12-29 10:57:33  PASSED                                                                   [100%]
```
and all results are in the performance dashboard

**Review by ebenahar — APPROVED** [review]

**Pull request merged**